modules/openapi-generator/src/main/resources/python/model_utils.mustache

```diff
@@ -1277,11 +1277,16 @@ def get_valid_classes_phrase(input_classes):
 def convert_js_args_to_python_args(fn):
     from functools import wraps
     @wraps(fn)
-    def wrapped_init(self, *args, **kwargs):
+    def wrapped_init(_self, *args, **kwargs):
+    """
+    An attribute named `self` received from the api will conflicts with the reserved `self`
+    parameter of a class method. During generation, `self` attributes are mapped
+    to `_self` in models. Here, we name `_self` instead of `self` to avoid conflicts.
```

Comment:
How about we stop mapping `self` to `_self` in generation?
That's what I was asking for when I said:
> And I think that we should remove the self word as a [reserved python word.](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonLegacyClientCodegen.java#L122) Then we can pass payload property self through to the init with no mutations and the java code will not rename self to _self.

Reply:
I don't agree it should still be reserved.
The problem would be moved elsewhere like in the `__init__` methods of the models.

https://github.com/OpenAPITools/openapi-generator/blob/ee2f585f37c16834f312e0e8440da777350f6560/modules/openapi-generator/src/main/resources/python/model_templates/method_init_simple.mustache#L11

And what if we have an attribute named `_self`. We are back to the same problem. Now, if the user has both `_self` and `self`, one will overwrite the other. I think it is reasonable to assume that it is less likely to happen.

 I think only modifying the argument in the wrapper is appropriate.

Reply:
Right, that's why I asked that we rename all selfs to _selfs, so we can treat the spec as the source of truth and not have a collision. But it sounds like we disagree about that. If you want to only change the wrapper and can get tests passing that is fine.

Reply:
Any idea why circleci doesn't pass?

Reply:
Yup
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   samples/openapi3/client/petstore/python/.openapi-generator/FILES
	modified:   samples/server/petstore/kotlin-server/ktor/src/main/kotlin/org/openapitools/server/apis/PetApi.kt
	modified:   samples/server/petstore/kotlin-server/ktor/src/main/kotlin/org/openapitools/server/apis/StoreApi.kt
	modified:   samples/server/petstore/kotlin-server/ktor/src/main/kotlin/org/openapitools/server/apis/UserApi.kt

no changes added to commit (use "git add" and/or "git commit -a")

This script runs in pull requests against the anticipated merge commit (as if the PR was merged now).
When you see unexpected files here, it likely means that there are newer commits in master that you need to 
rebase or merge into your branch.

Please run 'bin/utils/ensure-up-to-date' locally and commit changes (UNCOMMITTED CHANGES ERROR)
2
```
Can you remove the kotlin changes from this pr?